crates/transaction-emitter-lib/src/emitter/account_minter.rs

```diff
@@ -76,8 +76,6 @@ impl<'t> AccountMinter<'t> {
                 .unwrap()
                 .checked_add(
                     req.max_gas_per_txn * req.gas_price
-                        // for module publishing
-                        + 2 * req.max_gas_per_txn * req.gas_price * req.init_gas_price_multiplier,
```

Comment:
I've changed to always fund publishing modules from source account, to reduce the resource need for worker accounts (and we generally publish one module)